src/AspNetCoreModuleV2/AspNetCore/src/applicationinfo.cpp

```diff
@@ -358,60 +359,46 @@ APPLICATION_INFO::FindRequestHandlerAssembly(STRU& location)
 
 HRESULT
 APPLICATION_INFO::FindNativeAssemblyFromGlobalLocation(
-    PCWSTR libraryName,
-    STRU* struFilename)
+    PCWSTR pstrHandlerDllName,
```

Comment:
we need a mechanism to only log the error once. I will do it in my PR

Reply:
I'd say we centralize all logs to the event viewer.

Reply:
I meant this function would be called for every request in case of failure, (e.g., requesthandler folder was deleted by user) and thus caused log flooding. We should ensure this kind event logged once per application until some change (appoffline or configuration) happened.